conans/client/migrations.py

```diff
@@ -69,12 +80,15 @@ def _make_migrations(self, old_version):
         if old_version < Version("0.20"):
             if os.path.exists(os.path.join(self.client_cache.conan_folder, CONAN_CONF)):
                 raise ConanException("Migration: Your Conan version was too old, "
-                                     "please, remove the %s file manually, Conan will generate a new one." % CONAN_CONF)
+                                     "please, remove the %s file manually, Conan will generate "
+                                     "a new one." % CONAN_CONF)
 
         if old_version < Version("0.25"):
-            self.out.warn("Migration: Moving default settings from %s file to %s" % (CONAN_CONF, DEFAULT_PROFILE_NAME))
+            self.out.warn("Migration: Moving default settings from %s file to %s"
+                          % (CONAN_CONF, DEFAULT_PROFILE_NAME))
             conf_path = os.path.join(self.client_cache.conan_folder, CONAN_CONF)
-            default_profile_path = os.path.join(PROFILES_FOLDER, DEFAULT_PROFILE_NAME)
+            default_profile_path = os.path.join(self.client_cache.conan_folder, PROFILES_FOLDER,
```

Comment:
This missing folder was creating default profile locally in user folder, not in ``.conan``